x-pack/plugin/src/test/resources/rest-api-spec/api/ilm.move_to_step.json

```
@@ -1,6 +1,7 @@
 {
   "ilm.move_to_step": {
-    "documentation": "http://www.elastic.co/guide/en/index_lifecycle/current/index_lifecycle.html",
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/ilm-move-to-step.html",
+    "description": "Triggers execution of a specific step in the lifecycle policy. Since this action is potentially destructive, clients should not implement this API",
```

```suggestion
    "description": "Triggers execution of a specific step in the lifecycle policy. Since this action is designed to only be run in emergency situations, clients should not implement this API",
```

I think the wording needs to be changed here and I left a suggestion. The problem with the original wording is that we implement plenty of destructive APIs in the clients (all the delete APIs for example). The key here is that the API is not designed to be run programmatically because its only to be used in emergency situations to put a broken lifecycle back on track

---

ah, yes! we want to recommend implementing delete!